extract.go

```diff
@@ -78,6 +78,27 @@ func (e *Extractor) Values() (out []interface{}, err error) {
 	return
 }
 
+//ValuesFromTag returns an interface array with all the values of fields with the given tag
```

Comment:
Yes, this returns a slice of values. ```FiledValueFromTagMap()``` returns a map from field name to value (interface{})f